src/main/java/uk/gov/justice/digital/hmpps/whereabouts/dto/prisonApi/AmendmentDto.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.hmpps.whereabouts.dto.elite;
+package uk.gov.justice.digital.hmpps.whereabouts.dto.prisonApi;
```

Comment:
java packages should always be lowercase - see https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html